centreon/README.md

```diff
@@ -0,0 +1,63 @@
+Plugin for Centreon and Rudder
+==============================
+
+This project is part of Rudder - IT automation and compliance made easy.
+See: http://rudder-project.org for more information.
+
+Mailing list, irc : https://www.rudder-project.org/site/community/mailing-lists/
+
+Synposis
+--------
+
+This plugin aims at providing Rudder integration with Centreon. It can automatically
+add hosts to Centreon when the corresponding node is set up for monitoring in Rudder.
+
+The nodes can also have monitoring templates applied using the appropriate generic methods
+in the technique editor.
+
+Installation
+------------
+
+- Prerequisites :
+
+  - Install a Centreon server https://download.centreon.com
+    The server is not required to be running on the same host as the Rudder server.
+    However, this plugin has to be installed and running on the Rudder server.
+
+  - The Rudder server must have Python installed.
+
+- Generate the Rudder package : cd into the centreon directory, then make. The .rpkg package file
+  will be generated in the current directory.
+
+- Install the package on the Rudder server : /opt/rudder/bin/rudderpkg install-file <plugin>.rpkg
+
+- Edit the configuration file /opt/rudder/etc/centreon.conf. This config file has to contain your Centreon
+  server's webservice API URL, the credentials to access it, and the name of the Centreon poller used to 
+  monitor Rudder nodes (Central by default). You also need to provide the Rudder API URL, and a token to access
+  it, which you can issue from the Rudder server webapp or CLI.
```

Comment:
" from the Rudder server webapp or CLI" this is too generic, either say how to get it or don't say anything.
The best would be a pointer to a Rudder documentation url that already has the information.